eng/pipelines/runtime-official.yml

```diff
@@ -485,9 +633,9 @@ extends:
             - Build_tvossimulator_arm64_release_AllSubsets_Mono
             - Build_tvossimulator_x64_release_AllSubsets_Mono
             - Build_windows_x64_release_CrossAOT_Mono
-            - installer_coreclr__windows_x64_Release_
-            - installer_coreclr__windows_x86_Release_
-            - installer_coreclr__windows_arm64_Release_
+            - Build_windows_x64_release_AllSubsets_CoreCLR
```

Comment:
Now that we build like this, I don't think All subsets in the name really helps, it just lengthens the name and AzDO has a pretty narrow column for it. Can we just drop it?

Reply:
Agreed - it should be the default and legs that don't follow this pattern should have a display name describing their scenario.